spine-ios/Sources/Spine/Spine.Generated.swift

```diff
@@ -0,0 +1,3547 @@
+import Foundation
+import SpineWrapper
+
+public typealias SpineBlendMode = spine_blend_mode
+public typealias SpineMixBlend = spine_mix_blend
+public typealias SpineEventType = spine_event_type
+public typealias SpineAttachmentType = spine_attachment_type
+public typealias SpineConstraintType = spine_constraint_type
+public typealias SpineInherit = spine_inherit
+public typealias SpinePositionMode = spine_position_mode
+public typealias SpineSpacingMode = spine_spacing_mode
+public typealias SpineRotateMode = spine_rotate_mode
+public typealias SpinePhysics = spine_physics
+
+public final class SpineTransformConstraintData {
```

Comment:
This is a Swift module, so there's no need to prefix `SpineTransformConstraintData` and other types with `Spine`.

For use in ObjC, we can use an annotation like

```
@objc(SpineTransformConstraintData)
class TransformConstraintData: NSObject {
    // Implementation
}
```

This way, Swift users do not have to state the prefix all the time, while ObjC users still get a unique name.

Not sure how this would work for `typealias`. I feel like we can keep that as is with the `Spine` prefix, e.g. `SpineBlendMode`.

Reply:
We can only annotate classes that subclass `NSObject` with `@objc`. We could do this in a separate PR.